code/web/src/setup/routes/survey.js

```diff
@@ -4,7 +4,7 @@ import List from '../../modules/survey/Survey/List'
 // Crate routes
 export default {
   survey: {
-    path: '/survey',
+    path: (page = ':page') => (`/survey/${ page }`),
```

Comment:
interesting, 
what is the ':page'

Reply:
This is used for dynamic routing.

`page = ':page' is a default for a parameter.

':page' is a string which will be the default route ====> `/survey/:page`